runtime/vm/jfr.cpp

```diff
@@ -965,16 +965,23 @@ jfrCPULoad(J9VMThread *currentThread)
 				jfrEvent->jvmSystem = 0;
 			} else {
 				int64_t timeDelta = currentTime - jfrState->prevProcTimestamp;
-				jfrEvent->jvmUser = OMR_MIN((currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / ((double)numberOfCpus * timeDelta), 1.0);
-				jfrEvent->jvmSystem = OMR_MIN((currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / ((double)numberOfCpus * timeDelta), 1.0);
+				float timeElapsedAllCores = (float)numberOfCpus * timeDelta;
+				float userTime = (currentProcCPUTimes._userTime - jfrState->prevProcCPUTimes._userTime) / timeElapsedAllCores;
+				float systemTime = (currentProcCPUTimes._systemTime - jfrState->prevProcCPUTimes._systemTime) / timeElapsedAllCores;
+
+				jfrEvent->jvmUser = OMR_MIN(userTime, 1.0f);
+				jfrEvent->jvmSystem = OMR_MIN(systemTime, 1.0f);
 			}
 			jfrState->prevProcCPUTimes = currentProcCPUTimes;
 			jfrState->prevProcTimestamp = currentTime;
 
 			if (-1 == jfrState->prevSysCPUTime.timestamp) {
 				jfrEvent->machineTotal = 0;
 			} else {
-				jfrEvent->machineTotal = OMR_MIN((currentSysCPUTime.cpuTime - jfrState->prevSysCPUTime.cpuTime) / ((double)numberOfCpus * (currentSysCPUTime.timestamp - jfrState->prevSysCPUTime.timestamp)), 1.0);
+				float machineTotal = 
+					(currentSysCPUTime.cpuTime - jfrState->prevSysCPUTime.cpuTime) /
+					((float)numberOfCpus * (currentSysCPUTime.timestamp - jfrState->prevSysCPUTime.timestamp));
```

Comment:
Lines should be split with a binary operator beginning the continued line:
```c
				float machineTotal =
						(currentSysCPUTime.cpuTime - jfrState->prevSysCPUTime.cpuTime)
						/ ((float)numberOfCpus
						* (currentSysCPUTime.timestamp - jfrState->prevSysCPUTime.timestamp));
```
Continuations should be indented two tabs more than the first line.